{"schemaVersion":{"minor":3,"patch":0,"major":0},"sections":[],"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"sdkInitialisationInProgress","kind":"identifier"}]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError"]]},"abstract":[{"text":"If SDK initialisation is in progress and you try to initialise it again, this error will be thrown.","type":"text"}],"metadata":{"title":"CloudXError.sdkInitialisationInProgress","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"sdkInitialisationInProgress"}],"roleHeading":"Case","externalID":"s:10CloudXCore0A6XErrorO27sdkInitialisationInProgressyA2CmF","role":"symbol","modules":[{"name":"CloudXCore"}],"symbolKind":"case"},"kind":"symbol","identifier":{"url":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/sdkInitialisationInProgress","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/cloudxcore\/cloudxerror\/sdkinitialisationinprogress"]}],"references":{"doc://com.cloudx.sdk.core/documentation/CloudXCore":{"abstract":[],"type":"topic","url":"\/documentation\/cloudxcore","title":"CloudXCore","kind":"symbol","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","role":"collection"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError/sdkInitialisationInProgress":{"url":"\/documentation\/cloudxcore\/cloudxerror\/sdkinitialisationinprogress","kind":"symbol","type":"topic","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/sdkInitialisationInProgress","abstract":[{"type":"text","text":"If SDK initialisation is in progress and you try to initialise it again, this error will be thrown."}],"role":"symbol","title":"CloudXError.sdkInitialisationInProgress","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"sdkInitialisationInProgress"}]},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError":{"type":"topic","title":"CloudXError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"CloudXError"}],"navigatorTitle":[{"kind":"identifier","text":"CloudXError"}],"url":"\/documentation\/cloudxcore\/cloudxerror","abstract":[{"type":"text","text":"An error type that represents the CloudX SDK errors."}],"kind":"symbol","role":"symbol","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError"}}}